src/index.coffee

```diff
@@ -11,6 +11,6 @@ module.exports = (app, options) ->
   if !options or options.enabled isnt false
     autoMigrate = (options and options.migration) or 'auto-update'
     autoMigrate = require './' + autoMigrate
-    autoMigrate(app, options)
+    app.autoMigrateDone = autoMigrate(app, options).then -> debug "Done"
```

Comment:
Yeah, it the distinction `loaded`/`done` is nicer, but its best if they are an object `{done, loaded}` of promises - do you want me to make another PR ?